index.html

```diff
@@ -1420,13 +1420,13 @@ <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
         </p>
         <p>The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">Securing
             Mechanisms</a> apply to securing
-            <code>application/vc-ld+jwt</code> and
-            <code>application/vp-ld+jwt</code>,
-            <code>application/vc-ld+sd-jwt</code> and
-            <code>application/vp-ld+sd-jwt</code>,
+            <code>application/vc+jwt</code> and
+            <code>application/vp+jwt</code>,
+            <code>application/vc+sd-jwt</code> and
```

Comment:
```suggestion
            <code>application/vc-ld+sd-jwt</code> and
```

Reply:
Add this back for now.

Reply:
The Mike who likes consistency would prefer that we keep the media types in this specification as consistent as possible.  Specifically, I suggest that we stay with application/{vc-ld,vp-ld}+{jwt,sd-jwt,cose}.  Deviating from that pattern will only cause more confusion and probably inadvertent interop errors.